Cargo.lock

```diff
@@ -1966,9 +1966,9 @@ dependencies = [
 
 [[package]]
 name = "lsp-types"
-version = "0.92.0"
+version = "0.92.1"
```

Comment:
Why was the lockfile updated?

Reply:
I was getting local build errors with `0.92.0` after adding `#[derive(Default)]` for the new `DebugFlags` struct. updating to `0.92.1` fixed this.